scalapb-runtime-grpc/src/main/scala/com/trueaccord/scalapb/grpc/ProtoUtils.scala

```diff
@@ -0,0 +1,25 @@
+package com.trueaccord.scalapb.grpc
+
+import com.google.protobuf.InvalidProtocolBufferException
+import com.trueaccord.scalapb.GeneratedMessage
+import io.grpc.Metadata
+
+object ProtoUtils {
+  def metadataMarshaller[T <: GeneratedMessage](instance: T): Metadata.BinaryMarshaller[T] =
```

Comment:
Will do using implicit companion.

About ScalaPbMetadataMarshaller, I am afraid I don't follow it, can you elaborate ?